vendor/engines/saas/app/jobs/saas/trials_create_job.rb

```diff
@@ -0,0 +1,10 @@
+module Saas
+  class TrialsCreateJob < ActiveJob::Base
+
+    def perform(params)
+      user = params['current_user']['login']
+      params['url'] = "/settings/#{user}/trialed"
+      Analytics::PageJob.perform_later(params)
+    end
```

Comment:
It's /settings/{login}/trialed for both, I was under the assumption we want to track the subscriber/owner